package.json

```diff
@@ -69,6 +69,7 @@
         "@types/jest": "^24.9.1",
         "@types/lodash.debounce": "^4.0.6",
         "@types/node": "^12.12.26",
+        "@types/node-gettext": "^2.0.1",
```

Comment:
I think node-gettext and gettext-parser are also not needed anymore (they are for the same functionality as gettext-extractor was used for).

Reply:
i18n.tsx uses it…

Reply:
That also uses gettext-parser by the way…

Reply:
ah right, we should leave those in then (and only remove gettext-extractor).